orbit/cmd/orbit/orbit.go

```diff
@@ -418,6 +418,18 @@ func main() {
 				return err
 			}
 
+			// Get current version of osquery
+			osquerydPath, err = updater.ExecutableLocalPath("osqueryd")
+			if err != nil {
+				log.Info().Err(err).Msg("Could not find local osqueryd executable")
+			} else {
+				version := update.GetVersion(osquerydPath)
+				if version != "" {
+					log.Info().Msgf("Found osquery version: %s", version)
+					updateRunner.OsqueryVersion = version
+				}
+			}
```

Comment:
This should be executed after the if in line `492` (where you already have `osquerydPath` available)

Reply:
It is here because I need the osquery version before `updateRunner.UpdateAction()` because the update action may end up updating osquery.

Reply:
Mhm... but this change may break https://github.com/fleetdm/fleet/issues/13825 (There's no automation in Orbit so we rely on manual testing.)

Reply:
My code shouldn't break anything -- if something goes wrong in my code, `OsqueryVersion` simply ends up being empty. Is the issue that I'm using the `osquerydPath` variable? I can use another unique variable instead.